### PR TITLE
libvirtd: Add case to test qemu monitor socket

### PR DESCRIPTION
**Pull request by cliping**
XXXX-37948 - [libvirtd] Start vm - with qemu monitor socket creation delay

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

 (1/1) type_specific.io-github-autotest-libvirt.daemon.qemu_monitor_socket_creation_delay: PASS (25.46 s)

File: libvirt/tests/src/daemon/qemu_monitor_socket_creation_delay.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    """
+    Write a wrapper for qemu to wait several seconds (>3) before starts.
+
+    """
+    vm_name = params.get('main_vm')
+    vm = env.get_vm(vm_name)
+    emulator_dict = eval(params.get("emulator_dict"))
+    qemu_wrapper_path = params.get("qemu_wrapper_path")
+    qemu_wrapper = """#!/usr/bin/env python
+import os
```
Comment (chunfuwen) [inline libvirt/tests/src/daemon/qemu_monitor_socket_creation_delay.py:21]: maybe put those text into cfg files

File: libvirt/tests/src/daemon/qemu_monitor_socket_creation_delay.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+def run(test, params, env):
+    """
+    Write a wrapper for qemu to wait several seconds (>3) before starts.
+
+    """
+    vm_name = params.get('main_vm')
+    vm = env.get_vm(vm_name)
+    emulator_dict = eval(params.get("emulator_dict"))
+    qemu_wrapper_path = params.get("qemu_wrapper_path")
+    qemu_wrapper = """#!/usr/bin/env python
```
Comment (Yingshun) [inline libvirt/tests/src/daemon/qemu_monitor_socket_creation_delay.py:20]: What about moving this script to 'tp-libvirt/libvirt/tests/deps'?

Comment (smitterl) [reply libvirt/tests/src/daemon/qemu_monitor_socket_creation_delay.py:20]: +1

File: libvirt/tests/src/daemon/qemu_monitor_socket_creation_delay.py (hunk 1 above)
Comment (cliping) [reply libvirt/tests/src/daemon/qemu_monitor_socket_creation_delay.py]: Move to 'tp-libvirt/libvirt/tests/deps'. Thanks.

File: libvirt/tests/src/daemon/qemu_monitor_socket_creation_delay.py (hunk 2 above)
Comment (cliping) [reply libvirt/tests/src/daemon/qemu_monitor_socket_creation_delay.py]: Updated, thanks.

**Comment by Yingshun** [comment]
@cliping Is qemu_wrapper.py missing from this pr? Or any dependencies? I can't find it here.

**Comment by cliping** [comment]
> @cliping Is qemu_wrapper.py missing from this pr? Or any dependencies? I can't find it here.

Upload, thanks.

**Review by Yingshun — APPROVED** [review]

**Review by smitterl — APPROVED** [review]

**Review by chunfuwen — APPROVED** [review]
lgtm

**Pull request merged**